e2e/pod.go

```diff
@@ -212,7 +212,8 @@ func deletePodWithLabel(label, ns string, skipNotFound bool) error {
 }
 
 // calculateSHA512sum returns the sha512sum of a file inside a pod.
-func calculateSHA512sum(f *framework.Framework, app *v1.Pod, filePath string, opt *metav1.ListOptions) (string, error) {
+func calculateSHA512sum(f *framework.Framework, app *v1.Pod, opt *metav1.ListOptions) (string, error) {
+	filePath := app.Spec.Containers[0].VolumeMounts[0].MountPath + "/test"
```

Comment:
Maybe pass the file name as an argument so that it can become configurable?

Reply:
> Also, the PR description mentions calculating `md5sum` which doesn't seem to be done in the PR. Can you please change that too?

Yeah sure. Thanks for pointing out.

Reply:
I hope you are clear with, why I have not used filepath as argument?

Reply:
No. I might have missed it, did you explain it somewhere here?

Reply:
No, I thought in the later part of code you must have got the idea. So the jist here is, While calculating the the checksum for the clone we do not have app as a pointer and hence calculating the filepath was not easy rather I think calculating the filepath inside the function will be better. Second repeating the same filepath and passing it in the function was repeatative and hence, it will reduce the lines of code.

Reply:
please make file-path as the argument or else we will be asking the user to write the data to the same file (which doesn't seem correct)